src/planscape/planning/tests/helpers.py

```diff
@@ -49,3 +50,28 @@ def _create_scenario(
     scenario_result.save()
 
     return scenario
+
+
+# this is in the migration, but it all gets removed whenever tests TearDown
+# should we DRY this up and call it from collaboration utils
+def reset_permissions():
+    viewer = ["view_planningarea", "view_scenario"]
```

Comment:
Is there a more Django-y way to deal with data that's inserted from migrations, but which get wiped in the setUp/tearDown of a unit test?